cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1388,28 +1388,28 @@ def test_advanced_problems(self):
         Test the handling of advanced problem templates.
         """
         problem_templates = self.get_templates_of_type('problem')
-        ora_template = self.get_template(problem_templates, u'Peer Assessment')
-        self.assertIsNotNone(ora_template)
-        self.assertEqual(ora_template.get('category'), 'openassessment')
-        self.assertIsNone(ora_template.get('boilerplate_name', None))
+        circuit_template = self.get_template(problem_templates, u'Circuit Schematic Builder')
```

Comment:
I actually agree with you - the reason I changed it to "circuit" was to maintain the basic point of the test, while also assuring this gets a second look in the near future, as I do imagine the schematic builder will spin out into an xblock.